roles/network_plugin/calico/tasks/install.yml

```diff
@@ -4,6 +4,8 @@
     src: "cni-calico.conflist.j2"
     dest: "/etc/cni/net.d/{% if calico_version is version('v3.3.0', '>=') %}calico.conflist.template{% else %}10-calico.conflist{% endif %}"
     owner: kube
+  register: calico_conflist
+  notify: reset_calico_cni
```

Comment:
Will this trigger a reset even if no change applied to the conflist file?

Reply:
no, the `notify` is only applied when there's a change in the conflist file.